CSS/style.css

```diff
@@ -0,0 +1,759 @@
+    }
+    .pricing-card2{
+      flex: 1;
+        background-color: #151515;
+        padding: 2.5rem;
+        transition: all 1s ease;
+        border: 1px solid #4998FF;
+  
+         &:hover{
+        border: 1px solid #4998FF;
+        transform: scale(1.04);
+      }
+  
+        small{
+          text-transform: uppercase;
+          
+          p{
+            text-align: center;
+            font-weight: 600;
+            font-size: 20px;
+            padding: 10px 0;
+          }
+  
+        }
+        .price-tag{
+          background-color: #4998FF;
+          text-align: center;
+          width: 150px;
+          font-weight: 700;
+          color: #000;
+          font-size: 35px;
+          margin: 0.2rem auto;
+          padding: 5px 1rem;
+          border-radius: 5px;
+  
+          span{
+            font-size: 13px;
+          }
+        }
+  
+        div{
+          display: flex;
+          align-items: center;
+          justify-content: space-between;
+          padding: 1rem 0;
+          border-bottom: 0.5px solid gray;
+  
+          p{
+            color: #4998FF;
+          }
+        }
+  
+        .btn{
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          border: none;
+  
+          button{
+            padding: 10px 2rem;
+            font-weight: 600;
+            color: #fff;
+            background-color: #000;
+            border: 1px solid #4998FF;
+            border-radius: 50px;
+          }
+        }
+        
+        
+      }
+    }
+    .pricing-card3{
+      flex: 1;
+      background-color: #151515;
+      padding: 1.5rem;
+      transition: all 1s ease;
+
+       &:hover{
+      border: 1px solid #4998FF;
+      transform: scale(1.04);
+    }
+
+      small{
+        text-transform: uppercase;
+        
+        p{
+          text-align: center;
+          font-weight: 600;
+          font-size: 20px;
+          padding: 10px 0;
+        }
+
+      }
+      .price-tag{
+        background-color: #4998FF;
+        text-align: center;
+        width: 150px;
+        font-weight: 700;
+        color: #000;
+        font-size: 35px;
+        margin: 0.2rem auto;
+        padding: 5px 1rem;
+        border-radius: 5px;
+
+        span{
+          font-size: 13px;
+        }
+      }
+
+      div{
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        padding: 1rem 0;
+        border-bottom: 0.5px solid gray;
+
+        p{
+          color: #4998FF;
+        }
+      }
+
+      .btn{
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        border: none;
+
+        button{
+          padding: 10px 2rem;
+          font-weight: 600;
+          color: #fff;
+          background-color: #000;
+          border: 1px solid #4998FF;
+          border-radius: 50px;
+        }
+      }
+      
+      
+    }
+  }
+
+@media only screen and (min-width: 8px){
+
+}
```

Comment:
Very good. The best practice of a good developer is styling starting with mobile, then you use media queries to style for tablet and desktop. So inspect in your browser, then start styling for mobile before moving to desktop. So move all your styling for desktop into media queries. And start the original styling with mobile. It makes it very easy. Start and you won't regret it.

Work on responsiveness for the navbar for mobile. Use a side menu or off-canvas to do that. 

Consider adding reasonable padding to the container of the trending section so it aligns well to the page fo both desktop and mobile. Same goes to the container for pricing section.

All the best.